docs/organizations/configuring-default-git-provider-integration-settings.md

```diff
@@ -4,9 +4,17 @@ You can configure the default settings that Codacy uses to integrate with your G
 
 To configure these default settings, go to the Git provider integration page for your organization on the left navigation sidebar.
 
-![Default Git provider integration settings](images/default-git-provider-settings.png)
+![Default Git provider integration settings](images/default-git-provider-settings.png)<!--TODO PLUTO-550 Update screenshot-->
 
-The organization-level Git provider integration settings define the defaults that Codacy applies to new repositories. You can then customize the settings for each individual repository, which depend on your Git provider, [GitHub](../repositories-configure/integrations/github-integration.md), [GitLab](../repositories-configure/integrations/gitlab-integration.md) or [Bitbucket](../repositories-configure/integrations/bitbucket-integration.md).
+The organization-level Git provider integration settings define the defaults that Codacy applies to new repositories. You can then [apply these setting to all your repositories](#apply-all) already added to Codacy, or customize the settings for each individual repository, which depend on your Git provider, [GitHub](../repositories-configure/integrations/github-integration.md), [GitLab](../repositories-configure/integrations/gitlab-integration.md) or [Bitbucket](../repositories-configure/integrations/bitbucket-integration.md).
+
+## Applying default settings to all repositories {: id="apply-all"}
+
+To ensure that all your repositories are configured with the default Git provider integrating settings defined for your organization, apply these settings all repositories.
+
+To apply the default settings to all repositories, click the button **Apply default to all repositories**.
```

Comment:
Let's simplify, since it's a quite self-explanatory button. :nerd_face: 

```suggestion
To ensure that all your repositories are configured with the default Git provider integrating settings defined for your organization, click the button **Apply default to all repositories**.
```